.ci/openshift-ci/run_smoke_test.sh

```diff
@@ -40,13 +40,29 @@ hello_msg='Hello World'
 oc exec ${pod} -- sh -c "echo $hello_msg > $hello_file"
 
 info "Creating the service and route"
-oc apply -f ${script_dir}/smoke/service.yaml
-sleep 60
+if oc apply -f ${script_dir}/smoke/service.yaml; then
```

Comment:
The service API (https://kubernetes.io/docs/concepts/services-networking/service/) should work with kcli too. Maybe there is a firewall blocking the port 80 on the VM?

Reply:
Services are supported, but routes are not. We can use ingress, but that one requires ingress controller, so using another pod seems simpler for the purpose of testing pod execution.

Reply:
OK I found another way using service of type "NodePort", let me know which way you prefer.